docs/rfc/rfc-10.adoc

```diff
@@ -0,0 +1,248 @@
+:toc: macro
+
+= RFC 9: Solana tBTC
+
+:icons: font
+:numbered:
+toc::[]
+
+== Background
+
+There is a clear massive demand funnel present in Solana ecosystem for tBTC.
+This RFC aims at providing a bespoke user experience for minting tBTC on Solana.
+
+=== Current functionality
+
+Even today it is possible to bridge tBTC to Solana with Wormhole. The whole user
+experience is far from perfect, though. 
+
+First, the user has to deposit their BTC using Threshold Dashboard on Ethereum.
+This requires executing one Ethereum transaction. Next, once tBTC is minted on
+Ethereum, the user has to go to the Wormhole Token Bridge portal and bridge
+their Ethereum tBTC to Solana. This requires several transactions - one
+transaction on Ethereum and several transactions on Solana. 
+
+There is a significant cognitive effort on the user's side. Users are required
+to use two applications and execute transactions on three networks: Bitcoin,
+Ethereum, and Solana between them. Moreover, there is a period of time between
+the time BTC was deposited and the time tBTC is minted on Ethereum that requires
+the user to wait idly before they can bridge their Ethereum tBTC to Solana.
+
+== Proposal
+
+=== Goal
+
+Users should mint tBTC on Solana in the Threshold Dashboard with two
```

Comment:
> "mint tBTC on Solana"
Correct me if I'm wrong, but this means that there will be deployed a new canonical tBTC on Solana, right? Which also inherits from L2TBTC.sol? However, if the canonical tBTC on Solana is going to be a `wormholeTBTC`, then do we still need to mint it?

Reply:
Negative. We may deploy a canonical tBTC and we may not deploy it and consider wormhole tBTC as the canonical one. This sentence is just an abstract statement - we want users to mint their tBTC on Solana with the same experience as they have on Ethereum. The existence or non-existence of a gateway is just a technical detail. In the minimum version, the gateway is not required.